test-suite/shared/src/test/require-jdk8/org/scalajs/testsuite/javalib/time/MonthDayTest.scala

```diff
@@ -0,0 +1,218 @@
+package org.scalajs.testsuite.javalib.time
+
+import java.time.chrono.IsoChronology
+import java.time.temporal.{UnsupportedTemporalTypeException, ValueRange, ChronoField}
+import java.time.{DateTimeException, LocalDate, Month, MonthDay}
+
+import org.junit.Assert._
+import org.junit.Test
+import org.scalajs.testsuite.utils.AssertThrows._
+
+/**
+  * Created by alonsodomin on 22/12/2015.
+  */
+class MonthDayTest extends TemporalAccessorTest {
+  import ChronoField._
+  import MonthDay._
+  import DateTimeTestUtil._
+
+  final val min = of(Month.JANUARY, 1)
+  final val max = of(Month.DECEMBER, 31)
+  final val leapMonth = of(Month.FEBRUARY, 29)
+
+  val samples = Month.values().map(month => (month, month.minLength(), month.maxLength())).flatMap {
+    case (month, minDay, maxDay) =>
+      if (minDay != maxDay) Seq(of(month, 1), of(month, minDay), of(month, maxDay))
+      else Seq(of(month, 1), of(month, minDay))
+  }.toSeq
+
+  val yearSamples = 2000 to 2020
+  val leapYearLifted = (year: Int) => IsoChronology.INSTANCE.isLeapYear(year)
+
+  override def isSupported(field: ChronoField): Boolean =
+    field == ChronoField.MONTH_OF_YEAR || field == ChronoField.DAY_OF_MONTH
+
+  @Test override def test_range(): Unit = {
```

Comment:
You should probably use this implementation of `test_range` in `TemporalAccessorTest` rather than overriding it. Or a version that tests all of these cases.